docs/src/content/docs/guides/examples.mdx

```diff
@@ -0,0 +1,308 @@
+---
+title: Examples
+description: Learn through practical examples of using Clack
+---
+
+This guide provides practical examples of using Clack in different scenarios.
+
+## Basic Examples
+
+### Simple Text Input
+
+```ts twoslash
+import { text } from '@clack/prompts';
+
+const name = await text({
+  message: 'What is your name?',
+  placeholder: 'John Doe',
+}) as string;
```

Comment:
is it not a string by default? what does `text` resolve to?

Reply:
it is, but if you use `isCancel`, it should narrow to `string` i think

Reply:
i had to force the type cast, it wasn't able to infer it for some reason 😭